pkg/didmethod/peer/store.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package peerdid
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"errors"
```

Comment:
should be using an error value package.

we typically have used github.com/pkg/errors but xerrors might make more sense now.

Reply:
https://github.com/pkg/errors/issues/202

Reply:
@troyronda added xerrors package for errors - https://github.com/hyperledger/aries-framework-go/compare/2fb4cd0596e90bfd4f0dc59869ec454735c6df32..39c2c2a0932cb370c548269d68e7f638382d93a0